docs/source/intervention_models/lipid_based_nutrient_supplements/index.rst

```diff
@@ -87,82 +87,93 @@ more than 280 kcal/day (> 90 g dose).LNS are nutrient dense, require no cooking
-packets of vitamins and minerals in powder form that can be sprinkled
-onto any ready to eat semi-solid food consumed at home,
+Alternative recipes and formulations, other than LNS, are currently being explored using cereals mixed with other ingredients, including
+whey, soy protein isolate, dried skimmed milk, and sesame, cashew and chickpea paste, among others. These are
+fortified with vitamins and minerals and are commonly called **fortified blended foods (FBF)**. An example of a commonly used FBF
+is **corn soy blend plus (CSB ++)**, which is a cooked blend of milled, heat-treated corn and soybeans that is fortified with a vitamin and
+mineral premix. **Multiple micronutrient powders (MNP)** are also an alternative way of providing micronutrients. These are single-dose
+packets of vitamins and minerals in powder form that can be sprinkled onto any ready to eat semi-solid food consumed at home,
 school or any other point of use. [DAS_Cochrane_Review_2019]_
 
-.. todo::
 
-   Add a general narrative overview of the intervention, including what it is, what outcomes it affects, if/how/when/where it has been used, etc.
+:underline:`Description of intervention`
 
-.. todo::
+The intervention is the supplementation of children from aged **6 months to 23 months** with **LNS + complementary feeding** (intervention) compared with no intervention (control). The setting of the intervention is the community. 
+
+.. note::
 
-  Fill out the following table with a list of known outcomes affected by the intervention, regardless of if they will be included in the simulation model or not, as it is important to recognize potential unmodeled effects of the intervention and note them as limitations as applicable.
+  - Most studies included children aged six months to 18 months (Adu- Afarwuah 2007; Adu-Afarwuah 2016; Ashorn 2015; Bisimwa 2012; Christian 2015; Hess 2015; Iannotti 2014; Kumwenda 2014; Mangani 2015; Matias 2017; Phuka 2008; Siega-Riz 2014). 
+  - Four studies included children aged six to 24 months (Dewey 2017; Olney 2018; Luby   2018; Null 2018) and 
+  - one study included children aged six to 36 months (Huybregts 2012). 
+  - Four included studies enrolled pregnant women and provided lipid-based nutrient supplements (LNS) plus complementary feeding during pregnancy and post-partum, followed by infant supplementation at six months of age (Adu-Afarwuah 2016; Ashorn 2015; Dewey 2017; Olney 2018). However, Dewey 2017 had an intervention arm in which only children were supplemented, hence we only used the data from that arm in the analysis in this review. The other studies provided LNS plus complementary feeding to children after six months of age. * The review conducted a sensitivity analysis by removing trials that also supplemented pregnant women with LNS in addition to children (Adu-Afarwuah 2016; Ashorn 2015), and found no significant difference in the outcome (Analysis 1.3; Analysis 1.4 in Das DAS_Cochrane_Review_2019).
```

Comment:
What's with the asterisk before the last sentence?

> "... after six months of age. * The review conducted a sensitivity analysis..."

Reply:
oh...I just wanted to star that sentence to highlight it...

Reply:
Oh ok, no problem.  I just wasn't sure if it was a .rst formatting error or something.